flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/FlinkPlannerImpl.scala

```diff
@@ -68,12 +68,16 @@ class FlinkPlannerImpl(
     }
   }
 
-  @throws(classOf[SqlParseException])
+  @throws(classOf[FlinkSqlParseException])
```

Comment:
Do you mean `FlinkSqlParseException` here?  
Yeah, you are right, we should remove it from throws.